contributing/deployment-tools.txt

```diff
@@ -0,0 +1,132 @@
+OME deployment tools
+====================
+
+This section describes deployment tools supported by the OME team.
+It is primarily designed for the core OME developers who want to
+bring new or upgrade existing prerequisites. The following steps explain
+the connections between basic repositories and the testing workflow.
+
+Prerequisites locations
+-----------------------
+
+List of OME prerequisites is stored in multiple git repositories, each of 
+which is available from several locations.
+
+Infrastructure (Experimental)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Infrastructure is provided to simplify deployment using 
+`Ansible Tower<https://www.ansible.com>`.
```

Comment:
Just `Ansible`, `Ansible Tower` is their enterprise management wrapper around Ansible (which they might be opensourcing soon)